app/styles/application-launcher.less

```diff
@@ -0,0 +1,51 @@
+//
+// Application Launcher
+// --------------------------------------------------
+
+.navbar-pf,.navbar-pf-alt {
+  .applauncher-pf {
+    .applauncher-pf-icon {
```

Comment:
Instead of

```less
.applauncher-pf {
  .applauncher-pf-icon {
     ...
  }
}
```

Let's use

```less
.applauncher-pf .applauncher-pf-icon {
  ...
}
```